dashboard/config/application.rb

```diff
@@ -100,8 +100,6 @@ class Application < Rails::Application
       jquery.handsontable.full.css
       jquery.handsontable.full.js
     )
-    config.react.variant = :development
```

Comment:
NODE_ENV is different from the Chef node environment stuff (a Chef node is not Node.js Node).

On react-rails vs babel, I had an unmerged PR a while back with some relevant discussion. I will look for the link